### ISSUE TITLE
Elementor editor doesn't start after upgrading to 2.0.1, only in one page 

### ISSUE THREAD

**Issue by stratboy** [pojome/elementor#3934]
Hi, strangely enough, after upgrading I'm not able to edit a specific page with elementor. I mean, all pages are editable but one. On that one, Elementor is going to hang on the glowing E logo. These are my settings, and on the bottom you can find some debug errors:

```
== Server Environment ==
	Operating System: Darwin
	Software: Apache
	MySQL version: 5.6.38
	PHP Version: 7.1.12
	PHP Max Input Vars: 3500
	PHP Max Post Size: 32M
	GD Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.4
	Site URL: []
	Home URL: []
	WP Multisite: No
	Max Upload Size: 32 MB
	Memory limit: 96M
	Permalink Structure: /%postname%/
	Language: it-IT
	Timezone: Europe/Rome
	Admin Email: email@reghellin.com
	Debug Mode: Active

== Theme ==
	Name: Triveneta
	Version: 1.0
	Author: Luca Reghellin
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: it_IT
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:59.0) Gecko/20100101 Firefox/59.0

== Active Plugins ==
	Advanced Custom Fields PRO
		Version: 5.6.9
		Author: Elliot Condon

	Custom Taxonomy Order NE
		Version: 2.9.3
		Author: Marcel Pol

	Elementor
		Version: 2.0.1
		Author: Elementor.com

	Elementor Pro
		Version: 1.15.5
		Author: Elementor.com

	Polylang
		Version: 2.3.4
		Author: Frédéric Demarle

	Post Duplicator
		Version: 2.20
		Author: Metaphor Creations

	Regenerate Thumbnails
		Version: 3.0.2
		Author: Alex Mills (Viper007Bond)

	Simple Page Ordering
		Version: 2.3
		Author: Jake Goldman, 10up

	User Role Editor
		Version: 4.40.1
		Author: Vladimir Garagulya

	Yoast SEO
		Version: 7.1
		Author: Team Yoast


== Debug ==
	Errors: 
		TypeError: this.getRegion(...) is undefined
			at []/wp-content/plugins/elementor/assets/js/editor.js?ver=2.0.1 - 5363:10
			2018-03-28 08:31 +00:00
			x 4 times

		TypeError: c is undefined
			at []/wp-includes/js/jquery/ui/position.min.js?ver=1.11.4 - 11:295
			2018-03-28 08:29 +00:00

		TypeError: this.getRegion(...) is undefined
			at []/wp-content/plugins/elementor/assets/js/editor.js?ver=2.0.1 - 5363:10
			2018-03-28 08:29 +00:00
			x 10 times
```

**Comment by KingYes** [comment]
Duplicate https://github.com/pojome/elementor/issues/3926

**Comment by parfilov** [comment]
Some of my subscribers have the same troubles.
Other plugins are disabled, memory_limit — 256. No success.

**Comment by KingYes** [comment]
@parfilov @stratboy Will fixed in next version (2.0.2).

**Comment by stratboy** [comment]
@KingYes when will it be released?
